users/config.rst

```diff
@@ -1335,6 +1342,16 @@ be present in the ``defaults`` element:
     Even sharing with other remote devices can be done in the template by
     including the appropriate :opt:`folder.device` element underneath.
 
+.. option:: defaults.ignores
+
+    .. versionadded:: 1.19.0
+
+    Template for the :ref:`ignore patterns <ignoring-files>` applied to new
+    folders.  These are automatically copied to the ``.stignore`` file when a
+    folder is automatically accepted from a remote device.  The GUI uses them to
+    pre-fill the respective field when adding a new folder as well.  In XML,
+    each pattern line is represented as by a ``<line>`` element.
```

Comment:
Maybe those two scenarios (gui and auto-accept) are all there are, however I'd suggest just extending your first sentence to have a generic explanation which applies to everything (directly covers auto-accept, and the GUI behaviour is also just a direct consequence of this). 

```suggestion
    Template for the :ref:`ignore patterns <ignoring-files>`, which are applied to newly created folders (unless a `.stignore` file is already in place).
```

Reply:
I do think the difference matters.  The auto-accept part is done in the backend, while suggesting the default ignore patterns happens in the GUI by actively reading the defaults and submitting them after folder creation.  Since this is also the reference documentation for whoever uses the config API, they need to know that the second part will not happen unless they implement it.